api/resource/definitions/network/network.proto

```diff
@@ -18,6 +18,7 @@ message AddressSpecSpec {
   uint32 flags = 5;
   bool announce_with_arp = 6;
   talos.resource.definitions.enums.NetworkConfigLayer config_layer = 7;
+  uint32 priority = 8;
```

Comment:
Should we maybe adhere to the kernel (and IP stack generally) name `metric` here?

Reply:
kernel uses priority actually, and we already do use `priority` for the RouteSpec

Reply:
Okay